forms_pro/tests/test_invitations.py

```
@@ -0,0 +1,333 @@
+        team = frappe.get_doc(
+            {
+                "doctype": "FP Team",
+                "team_name": team_name or f"{self.fake.word()} Team",
+            }
+        )
+        team.insert()
+        self.teams_created.append(team.name)
+        frappe.set_user("Administrator")
+        return team.name
+
+    def _make_accepted_invitation(self, invitee_email: str, owner: str, team_id: str) -> object:
```

Ruff (0.15.4)

[warning] Unused method argument: `team_id`

(ARG002)